src/test/alert_tests.cpp

```diff
@@ -7,10 +7,13 @@
 //
 
 #include "alert.h"
+#include "chain.h"
+#include "chainparams.h"
 #include "clientversion.h"
 #include "data/alertTests.raw.h"
 
 #include "chainparams.h"
```

Comment:
As I wasn't allowed to sort this ;) just a quick nit, seems you now have a dup include of chainparams.h.